apps/nextjs/src/app/api/[...trpc]/route.ts

```diff
@@ -0,0 +1,21 @@
+import { createOpenApiFetchHandler } from "trpc-swagger/build/index.mjs";
+
+import { appRouter, createTRPCContext } from "@homarr/api";
+
+// Application Component || Define Handler
+// =================================================================================================
+// =================================================================================================
+const handler = (req: Request) => {
+  // Handle incoming swagger/openapi requests
+  return createOpenApiFetchHandler({
+    req,
+    endpoint: "/",
+    router: appRouter,
+    // createContext: () => { return {} },
+    createContext: () => createTRPCContext({ session: null, headers: req.headers }),
```

Comment:
Session can be read out of request when using `auth` wrapper

Reply:
I will implement auth keys in a later PR